configs/repos/jedi/packages/trans/package.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Trans(CMakePackage):
+    """Trans is the global spherical Harmonics transforms library, extracted from the IFS.
+    It is using a hybrid of MPI and OpenMP parallelisation strategies. The package contains
+    both single- and double precision Fortran libraries (trans_sp, trans_dp), as well as a
+    C interface to the double-precision version (transi_dp)."""
+
+    # DH* TODO UPDATE
+    homepage = "https://github.com/JCSDA-internal/trans"
+    git = "https://github.com/JCSDA-internal/trans.git"
+
+    maintainers = ['climbfuji']
+
+    version('develop', branch='develop', no_cache=True, preferred=True)
+    #version('bugfix-compile-failures', branch='bugfix/fix_compile_failures', no_cache=True)
+
+    variant('enable_mpi', default=True, description='Use MPI?')
+    variant('enable_omp', default=True, description='Use OpenMP?')
+
+    #variant('enable_double_precision', default=True, description='Enable double precision?')
+    #variant('enable_single_precision', default=True, description='Enable single precision?')
+
+    variant('enable_mkl',  default=True, description='Use MKL?')
```

Comment:
Note to self. Turn off MKL for now (then can remove the special package config for macos) to avoid problems with multiple specs of large parts of the software stack when building all applications, as observed on gaea and orion.